modules/pekko-http/docs/README.md

```diff
@@ -0,0 +1,34 @@
+# Pekko-Http module for PureConfig
```

Comment:
Nitpick: Can we spell this the same way [the project does](https://pekko.apache.org/docs/pekko-http/current/), i.e. "Pekko HTTP"?

```suggestion
# Pekko HTTP module for PureConfig
```